src/online/online.controller.ts

```diff
@@ -0,0 +1,42 @@
+import {
+    CacheInterceptor,
+    CacheTTL,
+    Controller,
+    Get,
+    Query,
+    UseInterceptors,
+} from '@nestjs/common';
+import {
+    ApiOkResponse,
+    ApiQuery,
+    ApiTags,
+} from '@nestjs/swagger';
+import { ATCInfo } from './atc-info.class';
+import { VatsimService } from './vatsim.service';
+
+@ApiTags('ONLINE')
+@Controller('online')
```

Comment:
We try to version new APIs. `api/v1/online`. Maybe the controller should be called ATC since online isn't very descriptive.

Reply:
The idea was to have an endpoint for everything related to online networks.
Is it ok for you to keep online ?

Reply:
I finally go for ATC controller which make more sense indeed